frontend/src/metabase/querying/utils/drills/compare-aggregations-drill/compare-aggregations-drill.tsx

```diff
@@ -0,0 +1,63 @@
+import { useDispatch } from "metabase/lib/redux";
+import { setUIControls } from "metabase/query_builder/actions";
+import {
+  CompareAggregations,
+  getTitle,
+} from "metabase/query_builder/components/CompareAggregations";
+import type {
+  ClickActionPopoverProps,
+  Drill,
+} from "metabase/visualizations/types/click-actions";
+import * as Lib from "metabase-lib";
+
+export const compareAggregationsDrill: Drill<
+  Lib.CompareAggregationsDrillThruInfo
+> = ({ question, query, stageIndex, clicked }) => {
+  if (!clicked.column || typeof clicked.column.aggregation_index !== "number") {
```

Comment:
I believe we need to use MBQB lib for that and do not compute the clause/column on our own. Other similar drills use `filterDrillDetails` to get the MBQL lib column. Can we do something similar here? E.g. we could add an `aggregationIndex` to `displayInfo` for this drill.

Reply:
Done: f88a55597bf9e2c3b6f147e4e7ed17666b0b1bce + d012855b073c4d29f992428d75d235c69cda9cfb + d7abad6ed8335109475746ea7f904b11d0b00b25